apps/judicial-system/web/src/routes/CourtOfAppeal/Result/Result.tsx

```diff
@@ -18,15 +18,13 @@ import Conclusion from '@island.is/judicial-system-web/src/components/Conclusion
 import CaseFilesOverview from '../components/CaseFilesOverview/CaseFilesOverview'
 import CourtOfAppealCaseOverviewHeader from '../components/CaseOverviewHeader/CaseOverviewHeader'
 
-import { courtOfAppealResult as strings } from './Result.strings'
-import { courtOfAppealRuling as rulingStrings } from '../Ruling/Ruling.strings'
-
 import { useIntl } from 'react-intl'
```

Comment:
Please consider ordering the imports - this import should be grouped with imports at the top of the file.

The general rule is to put third party imports at the top, then full path monorepo imports, then full path client imports and finally relative imports.